localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```
@@ -18,3 +19,186 @@
+ innovation_kodo_desc:0 "Tauren legends abound regarding the great horned kodos of Kalimdor’s Barrens. Some myths tell of rare kodos that are bound to the spirits of the sky and storm, thus allowing the colossal beasts to harness the powers of lightning itself. While kodos are generally considered gentle by nature, they are fierce opponents when threatened and have occasionally been known to devour their enemies whole. Our tamers have successfully learned how to communicate and tame these grand creatures."
+ 
+ innovation_marauder:0 "Marauders"
+ innovation_marauder_desc:0 "Like most of their kin, Marauders are savagery and brutality incarnate. Marauders are fast, aggressive and keen to show no mercy. Donning the hides of their enemies, Marauders are not known for taking prisoners."
+    
+ innovation_magi:0 "Magi"
+ innovation_magi_desc:0 "Ogre Magi are skilled spellcasting warriors. The two-headed ogre spellcasters are typically larger and more intelligent than their one-headed brethren. They often enjoy an elevated position among the ogre clans."
+ 
+ innovation_naga_ground:0 "Tidal Terrors"
+ innovation_naga_ground_desc:0 "With our unique ground armaments, we have produced regiments like the Myrmidons and the Sirens."
+ 
+ innovation_naga_taming:0 "Songs of Sorrow"
+ innovation_naga_taming_desc:0 "We have tamed the Couatl and enslaved the Sea Giants, and now we have incorporated them into our army."
+ 
+ innovation_savage:0 "Savages"
+ innovation_savage_desc:0 "Savages are fanatic and barbaric warriors of our clan. They are strong, stubborn, and always looking to spill some blood."
+ 
+ innovation_silithid:0 "Silithid"
+ innovation_silithid_desc:0 "Silithids are tough and sophisticated insectoid creatures. The chitin plates of their bone armor are intertwined in such a way that it makes their armor incredibly hard. Lumped in huge numbers, the silithids are devouring swarms. They stop for nothing but the demise of other life on Azeroth."
+
+ innovation_snow_hunter:0 "Snow Hunters"
+ innovation_snow_hunter_desc:0 "Our kind had to adapt to the elements in order to survive. The best of our warriors are the hunters who keep us safe."
+    
+ innovation_thornguard:0 "Thornguards"
+ innovation_thornguard_desc:0 "The Thornguards are the finest warriors from our tribe, who defend us from any outsiders. They are heavily armed and armored. They will stop any who dare to trespass or any who dare to not submit."
+
+ innovation_tidehunter:0 "Tidehunters"
+ innovation_tidehunter_desc:0 "Mglrmglmglmgl. Mmmm Mmmurlok. Uuua n Mrgllll glrrm gl!"
+ 
+ innovation_transcendent:0 "Transcendent Avatars"
+ innovation_transcendent_desc:0 "Pandaren revere transcendents as embodiments of wisdom: the spirit of the universe made flesh, residing in the shell of a pandaren host. These are the pandaren’s ultimate heroes: Spiritual avatars capable of summoning vast storms of lighting, smashing mountains with their terrible fists, and breathing winds into existence."
```

I admit, I'm more used to British English than American English, so I'm not 100% sure here, but I don't think spiritual should be capitalized here even if it is after a comma, as it is not starting an independent clause. @arithon, can you confirm?  
```suggestion
 innovation_transcendent_desc:0 "Pandaren revere transcendents as embodiments of wisdom: the spirit of the universe made flesh, residing in the shell of a pandaren host. These are the pandaren’s ultimate heroes: spiritual avatars capable of summoning vast storms of lighting, smashing mountains with their terrible fists, and breathing winds into existence."
```